dnf/cli/format.py

```diff
@@ -15,8 +15,23 @@
 # Red Hat, Inc.
 
 from __future__ import unicode_literals
+from datetime import datetime
```

Comment:
> It's simply against our code conventions; dot.

So isn't a time to change code conventions especially when the current code doesn't already meet them? :)